test/battle/move_effect/upper_hand.c

```diff
@@ -99,12 +99,12 @@ SINGLE_BATTLE_TEST("Upper Hand fails if the target moves first")
     }
 }
 
-SINGLE_BATTLE_TEST("Upper Hand is boosted by Sheer Force")
+SINGLE_BATTLE_TEST("Upper Hand is boosted by Sheer Force") // why does this move need a special test?
```

Comment:
It's because it can't be added together with all the other sheer force tests.

Should have asked in the dev channel. Now you have to waste time removing the comment. 😅

Reply:
Also conficts